paasta_tools/setup_tron_namespace.py

```diff
@@ -107,6 +108,22 @@ def main():
     failed = []
     skipped = []
 
+    master_config = tron_tools.create_complete_master_config(
+        cluster=args.cluster,
+        soa_dir=args.soa_dir,
+    )
+    if args.dry_run:
+        log.info(f"Would update {MASTER_NAMESPACE} to:")
+        log.info(f"{master_config}")
+        updated.append(MASTER_NAMESPACE)
+    else:
+        if client.update_namespace(MASTER_NAMESPACE, master_config):
```

Comment:
this is new code, not mentioned in PR title or description, did you mean to include it?

Reply:
I mentioned it in the title sorta.
The theory here is that we treat the master namespace as special case *only* in this script, and nowhere else. That reduces the complexity anywhere else.
I think it is OK for paasta not to support having jobs in the `MASTER` namespace anyway.